nfcreader/src/main/java/company/tap/nfcreader/open/reader/TapNfcCardReader.java

```diff
@@ -30,6 +38,12 @@ public TapNfcCardReader(Activity activity) {
         tapNfcUtils = new TapNfcUtils(activity);
         provider = new TapNfcProvider();
         logger = LoggerFactory.getLogger(TapNfcCardReader.class);
+        String app_name = activity.getApplicationInfo().loadLabel(activity.getPackageManager()).toString();
+        // Capture author info & user status
+        Map<String, String> parameters = new HashMap<String, String>();
+        parameters.put("sdkVersion", "1.0");
+        parameters.put("clientApp",app_name);
```

Comment:
`app_name` will always be the same, it's the caller package name which is the Card Form micros SDK 
Why to log something like this?

Reply:
No, app_name will be dynamically fetched from the client app name though its label. Which ever app or sdk uses this microSDK it will fetch the details.
For example, in current case it gets the name of the sample app which uses this library.

Reply:
> in current case it gets the name of the sample app which uses this library.

Exactly and the only user for this kit will be through our SDK not the merchant app, you are getting the package name of the caller activity which will always be our SDK activity not the merchant activity

Reply:
But in case if any other app , other than the in future wants to use this kit directly  than it will be helpful right?
If not needed will remove it. no issues.

Reply:
If we used it in other SDK, it will be the same idea we already know that we are using it, we actually don't care if it's goSell or goPay 
The analytics is used to track something we don't know and provide us with helpful informations